### PR TITLE
fix(tree): fix error cause by child node's data is not reactive

### LINKED ISSUE #1389: [bug report] tree component delayed loading, unable to expand by default
### Element Plus version
1.0.2-beta.30

### OS/Browsers version
Chrome 88 on macOS 11.1

### Vue version
3.0.5

### Reproduction Link
https://codepen.io/huijiewei/pen/XWNWdpY

### Steps to reproduce
https://codepen.io/huijiewei/pen/XWNWdpY

### What is Expected?
延迟加载无法默认展开

### What is actually happening?
展开的树节点一直加载中

### PR DESCRIPTION
**Pull request by Ryan2128**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

Fixed #1389

### CODE REVIEW

File: packages/tree/src/model/node.ts [hunk 1]
```diff
@@ -252,7 +255,8 @@ export default class Node {
         parent: this,
         store: this.store,
       })
-      child = new Node(child as TreeNodeOptions)
+      child = reactive(new Node(child as TreeNodeOptions)) as Node
+      (child as Node).initialize()
```
Comment (JeremyWuuuuu) [inline packages/tree/src/model/node.ts:259]: Why casting it twice?

File: packages/tree/src/model/tree-store.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {
   FilterValue,
   TreeNodeData,
 } from '../tree.type'
+import { isProxy } from 'vue'
```
Comment (JeremyWuuuuu) [inline packages/tree/src/model/tree-store.ts:14]: Remove this line since it was not referenced

File: packages/tree/src/model/node.ts (hunk 1 above)
Comment (Ryan2128) [reply packages/tree/src/model/node.ts]: Because IDE will throw a error, I will change it.

Comment (JeremyWuuuuu) [reply packages/tree/src/model/node.ts]: I think that was because you cast reactive<Node> to Node

**Review by zazzaz — APPROVED** [review]

**Pull request merged**